Source/Core/DolphinNoGUI/PlatformX11.cpp

```diff
@@ -46,7 +46,7 @@ class PlatformX11 : public Platform
 
   Display* m_display = nullptr;
   Window m_window = {};
-  Cursor m_blank_cursor = None;
+  Cursor m_blank_cursor = 0L;  // None
```

Comment:
This is some jank due to me needing to undefine `None` which `X.h` defines to be 0L, since that define interferes with some enums.  I'm not sure if there's a better workaround.